Update subscriptions.md - Updated FF Developer definition (#6594)
harness/developer-hub: docs/platform/get-started/subscriptions-licenses/subscriptions.md

The following Harness modules are directly involved in a developer's day-to-day workflow. These modules consume a developer license for every developer involved.

- **Code Repository (CODE):** All users on the Harness Platform, who have permission to access Harness CODE module are considered CODE Developers and consume a license.
- **Continuous Integration (CI):** All users on the Harness Platform, who have permission to access Harness CI module are considered CI Developers and consume a license.
- **Internal Developer Portal (IDP):** All users on the Harness Platform, who have permission to access Harness IDP module are considered IDP Developers and consume a license.

> - **Feature Flags (FF):** All Users on the Harness Platform, who have permission to create / update / delete Feature Flags and/or Delivery Groups, within the Harness FF module, will be considered as FF Developers and will consume a license.

Reviewer: ```suggestion
- **Feature Flags (FF):** All Users on the Harness Platform, who have permission to create / update / delete Feature Flags and/or Delivery Groups, within the Harness FF module, are considered FF Developers and consume a license.
```

<details>
<summary>Feature Flags (FF) Developers and Monthly Active Users (MAUs)</summary>

Harness Feature Flags is a feature flag rollout and management module that tracks **Developers** and **MAU** (Monthly Active Users) for license consumption.

> All Users on the Harness Platform, who have permission to create / update / delete Feature Flags and/or Delivery Groups, within the Harness FF module, will be considered as **FF Developers** and will consume a license.

Reviewer: ```suggestion
All Users on the Harness Platform, who have permission to create / update / delete Feature Flags and/or Delivery Groups, within the Harness FF module, are considered **FF Developers** and consume a license.
```
